test/clj_test_containers/core_test.clj

```diff
@@ -30,37 +32,37 @@
 
 
   (testing "Executing a command in the running Docker container with a custom container"
-    (let [container (init {:container (PostgreSQLContainer. "postgres:12.2")})
-          initialized-container (start! container)
-          result (execute-command! initialized-container ["whoami"])
-          stopped-container (stop! container)]
+    (let [container (sut/init {:container (PostgreSQLContainer. "postgres:12.2")})
+          initialized-container (sut/start! container)
+          result (sut/execute-command! initialized-container ["whoami"])
+          _stopped-container (sut/stop! container)]
       (is (= 0 (:exit-code result)))
       (is (= "root\n" (:stdout result))))))
 
 (deftest execute-command-in-container
 
   (testing "Executing a command in the running Docker container"
-    (let [container (create {:image-name "postgres:12.2"
-                             :exposed-ports [5432]
-                             :env-vars {"POSTGRES_PASSWORD" "pw"}})
-          initialized-container (start! container)
-          result (execute-command! initialized-container ["whoami"])
-          stopped-container (stop! container)]
+    (let [container (sut/create {:image-name "postgres:12.2"
+                                 :exposed-ports [5432]
+                                 :env-vars {"POSTGRES_PASSWORD" "pw"}})
+          initialized-container (sut/start! container)
+          result (sut/execute-command! initialized-container ["whoami"])
+          _stopped-container (sut/stop! container)]
```

Comment:
I have not seen a variable name starting with an underscore, is this a common convention? What does it imply?

Reply:
It implies that the variable is unused. In this case, the binding is here for clarity, but `_stopped-container` isn't read.